src/app/components/configure-app/configure-app.component.html

```diff
@@ -71,6 +72,23 @@ <h3 class="uk-card-title">Display Settings</h3>
             </div>
           </div>
 
+          <div class="uk-width-1-1">
+            <div class="uk-form-horizontal">
+
+              <div class="uk-margin">
+                <label class="uk-form-label" for="form-horizontal-select1">
+                  Hide Balances <span uk-icon="icon: info;" uk-tooltip title="Hide balances when viewing wallet and account details. Does not affect pending incoming amounts on the receive page."></span>
```

Comment:
the term "pending" has been replaced with "ready to receive" in UX, we shouldn't use it

Reply:
Done.